### ISSUE TITLE
Watch says "not available" and no debug info

### ISSUE THREAD

**Issue by JetLennit** [sanmont/vscode-kickass-studio#33]
When I try to add the expression "h:$c000[8]" it says "not available". It would also seem that there is no debug information in general. I believe I configured everything correctly, and given the software involved it doesn't seem the fact I'm on linux should cause any trouble.

**Comment by sanmont** [comment]
Hi Jet! Thank you for showing interest in kiskcass studio! I developed the
plugin I Linux so it shouldn't be a problem. On the weekend I'll take a
look and keep you posted!

**Comment by sanmont** [comment]
Hi @JetLennit  I tried it and it works correctly for me. Sorry if these are basic questions but just to rule out: 
- The program was compiled correctly? (it normally compiles to ./bin  but you can see which is the target in .vscode/launch.json in the property "binDirectory")
- Vice has launched and run the program?
- You have the program paused or on a breakpoint? If the program launched correcctly using vice it needs to be paused to show the memory contents and the registers.

**Comment by JetLennit** [comment]
@sanmont sometimes the basic questions are the most important! Last time I tried adding a breakpoint it didn't do anything but I figured I should retest just to make sure and it worked this time. Now everything works as expected! Thank you for your time, and thank you for this fantastic tool! Not sure why it didn't work last time and did this time, but I'll let you know if I have any further troubles.
